src/App/App.js

```diff
@@ -0,0 +1,83 @@
+import './App.css';
+import React, { Component } from 'react';
+import Header from '../Header/Header';
+import CardContainer from '../CardContainer/CardContainer';
+import MovieDisplay from '../MovieDisplay/MovieDisplay';
+// import MovieDetails from './MovieDetails/MovieDetails';
+// import movieData from '../data/dummyData';
+import { fetchData } from '../util.js'
+
+
+class App extends Component {
+ constructor(){
+   super();
+   this.state = {
+     movies: [],
+     currentSelectionId: null,
+     error: null,
+     dummyDisplay: {
+      id: 1, 
+      title: "Fake Movie Title", 
+      poster_path: "https://image.tmdb.org/t/p/original//7G2VvG1lU8q758uOqU6z2Ds0qpA.jpg", 
+      backdrop_path: "https://image.tmdb.org/t/p/original//oazPqs1z78LcIOFslbKtJLGlueo.jpg", 
+      release_date: "2019-12-04", 
+      overview: "Some overview that is full of buzzwords to attempt to entice you to watch this movie! Explosions! Drama! True love! Robots! A cute dog!", 
+      average_rating: 6, 
+      genres: ["Drama"], 
+      budget:63000000, 
+      revenue:100853753, 
+      runtime:139, 
+      tagline: "It's a movie!" 
+      }   
+   }
+ }
+
+ componentDidMount = () => {
+  fetchData('moves').then(data => {
+    this.setState({movies: [...this.state.movies, ...data.movies]})
+  }).catch((data) => {console.log("I made it to catch", data)
```

Comment:
We can think about a helper function here that creates some sort of jsx block OR passes the catch error as an argument to our error display component.

Reply:
In the `catch`, you can update state so that the `error` property holds a value. Then, use conditional rendering to show the error. Some people choose to make an Error component, others may choose to just render some text there.